content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-kubectl.md

```diff
@@ -23,118 +31,134 @@ description: Creating Secret objects using kubectl command line.
+used by Pods to access services. For example, you might need a Secret to store
+the username and password needed to access a database.
+-->
+`Secret` 对象用来存储敏感数据，如 Pod 用于访问服务的凭据。例如，为访问数据库，你可能需要一个
+Secret 来存储所需的用户名及密码。
 
-```shell
-echo -n 'admin' > ./username.txt
-echo -n '1f2d1e2e67df' > ./password.txt
-```
+<!-- 
+You can create the Secret by passing the raw data in the command, or by storing
+the credentials in files that you pass in the command. The following commands
+create a Secret that stores the username `admin` and the password `S!B\*d$zDsb=`.
+-->
+你可以通过在命令中传递原始数据，或将凭据存储文件中，然后再在命令行中创建 Secret。以下命令
+将创建一个存储用户名 `admin` 和密码 `S!B\*d$zDsb=` 的 Secret。
 
-<!--
-In these commands, the `-n` flag ensures that the generated files do not have
-an extra newline character at the end of the text. This is important because
-when `kubectl` reads a file and encodes the content into a base64 string, the
-extra newline character gets encoded too.
+<!-- 
+### Use raw data 
 -->
-在这些命令中，`-n` 标志确保生成的文件在文本末尾不包含额外的换行符。
-这一点很重要，因为当 `kubectl` 读取文件并将内容编码为 base64 字符串时，多余的换行符也会被编码。
+### 使用原始数据
 
-<!--
-The `kubectl create secret` command packages these files into a Secret and creates
-the object on the API server.
+<!-- 
+Run the following command: 
 -->
-`kubectl create secret` 命令将这些文件打包成一个 Secret 并在 API 服务器上创建对象。
+执行以下命令：
 
 ```shell
 kubectl create secret generic db-user-pass \
-  --from-file=./username.txt \
-  --from-file=./password.txt
+    --from-literal=username=devuser \
+    --from-literal=password='S!B\*d$zDsb='
 ```
 
-<!-- The output is similar to: -->
-输出类似于：
-
-```
-secret/db-user-pass created
-```
+<!-- 
+You must use single quotes `''` to escape special characters such as `$`, `\`,
+`*`, `=`, and `!` in your strings. If you don't, your shell will interpret these
+characters. 
+-->
+你必须使用单引号 `''` 转义字符串中的特殊字符，如 `$`、`\`、`*`、`=`和`!` 。否则，你的 shell 
+将会解析这些字符。
 
-<!--
-The default key name is the filename. You can optionally set the key name using
-`--from-file=[key=]source`. For example:
+<!-- 
+### Use source files 
 -->
-默认密钥名称是文件名。 你可以选择使用 `--from-file=[key=]source` 来设置密钥名称。例如：
+### 使用源文件
 
-```shell
-kubectl create secret generic db-user-pass \
-  --from-file=username=./username.txt \
-  --from-file=password=./password.txt
-```
+<!-- 
+1.  Store the credentials in files with the values encoded in base64: 
+-->
+1.  对凭证的取值作 base64 编码后保存到文件中：
 
-<!--
-You do not need to escape special characters in password strings that you 
-include in a file.
+    ```shell
+    echo -n 'admin' | base64 > ./username.txt
+    echo -n 'S!B\*d$zDsb=' | base64 > ./password.txt
+    ```
+<!-- 
+The `-n` flag ensures that the generated files do not have an extra newline
+character at the end of the text. This is important because when `kubectl`
+reads a file and encodes the content into a base64 string, the extra
+newline character gets encoded too. You do not need to escape special
+characters in strings that you include in a file. 
 -->
```

Comment:
These paragraphs are indented in the English source. Those indentations do matter because the file is in markdown format.

Please keep the indentation unchanged. When adding `<!--` and `-->`, make sure those lines are indented the same way has the commented paragraphs. When translating the text, the translation should be indented as well.